### DISCUSSION TITLE
Application size limit?

### DISCUSSION THREAD

**Discussion by Aiden007700** [beginner-corp/begin-community#70] [Support] [answered]
What is the Application size limit? I believe I saw that the max size of an app should not exceed 500Mb. Can someone confirm this?  

Is there any way to check how much our app weighs?  I am getting the following error: `npm WARN tar ENOSPC: no space left on device, write` however when I run `sudo du -sh` It looks like my app just weighs 429Mb.

**Comment by brianleroux** [comment, marked as answer]
hey @Aiden007700 Lambdas functions are currently limited to 500mb (but you can have more than one!)

**Reply by Aiden007700** [reply]
Is there any way to check the size of our app? When I run it locally I see that it weighs 429Mb so it should work, but I get the error message above, so clearly it weighs more than 500Mb and now I have conflicting information.

**Reply by brianleroux** [reply]
I use an app called DaisyDisk to stay on top of `node_modules`.

**Reply by ryanblock** [reply]
I believe that is a recommendation, not a requirement. You should be able to deploy a function that is that large, but know up front that doing so will impact performance. The smaller the cloud function, the better.
